### ISSUE TITLE
Crash when preparing payment request

### ISSUE THREAD

**Issue by matthmart** [Ingenico-ePayments/connect-sdk-client-swift#3]
We are trying to prepare a payment request using `Session.prepare()` method. But we have a crash at line 228, in the file `Encryptor.swift`: https://github.com/Ingenico-ePayments/connect-sdk-client-swift/blob/master/IngenicoConnectKit/Cryptography/Encryptor.swift#L228

The error is :

```
Thread 1: Fatal error: Unexpectedly found nil while unwrapping an Optional value
```

There is no error on the payment request after calling `validate()` method. The SDK version is 3.0.0. The session is created on the pre-production environment, using the Ogone endpoint.

Thank you.

**Comment by manhhavu** [comment]
I have the same problem with the new version 3.0.0:

<img width="1032" alt="crash" src="https://user-images.githubusercontent.com/4518211/48150411-e6e17900-e2be-11e8-89ff-47d0234d4f24.png">

**Comment by milanpanchal** [comment]
I'm still facing the same issue. Please fix the issue.

**Comment by rob-spoor** [comment]
This should be resolved soon. We cannot currently give an ETA though.

**Comment by LeonStemerdink** [comment]
This issue has been fixed in release 3.2.0.
